examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,340 @@
+# date is first argument that is passed to ``date_range()`` and it marks the first
+# data in the object ``x`` that will be plotted.
+
+x = pd.date_range("2018-03-01", periods=7, freq="BQS")
+y = [4, 5, 6, 8, 6, 3, 5]
+
+fig = pygmt.Figure()
+fig.plot(
+    projection="X10c/10c",
+    region=[datetime.datetime(2017, 12, 31), datetime.datetime(2019, 12, 31), 0, 10],
+    frame=["WSen", "ag"],
+    x=x,
+    y=y,
+    style="i0.4c",
+    pen="1p",
+    color="purple",
+)
+
+fig.show()
+
+########################################################################################
+# Using :meth:`xarray.DataArray`
+# -------------------------------------
+#
+# In this example, instead of using a ``pd.date_range`` object, ``x`` is initialized
+# as an ``xarray.DataArray`` object. Such object provide a wrapper around traditional
+# data formats allowing this data to have varrying labeled dimensions and support
+# operations that use various pieces of metadata. The following code uses a
+# ``pandas.date_range`` object to fill the DataArray with data, yet this is not
+# essential for the creation of a valid DataArray.
+
+x = xr.DataArray(data=pd.date_range(start="2020-01-01", periods=4, freq="Q"))
+y = [4, 7, 5, 6]
+
+fig = pygmt.Figure()
+fig.plot(
+    projection="X10c/10c",
+    region=[datetime.datetime(2020, 1, 1), datetime.datetime(2021, 4, 1), 0, 10],
+    frame=["WSen", "ag"],
+    x=x,
+    y=y,
+    style="n0.4c",
+    pen="1p",
+    color="red",
+)
+fig.show()
+
+# Code
+
+########################################################################################
+# Generating Region Using :meth:`pygmt.info`
+# ----------------------
+#
+# Explanation of supported parameters + bug at #597.
+
+data = [
+    ["20200712", 1000],
+    ["20200714", 1235],
+    ["20200716", 1336],
+    ["20200719", 1176],
+    ["20200721", 1573],
+    ["20200724", 1893],
+    ["20200729", 1634],
+]
+
+df = pd.DataFrame(data, columns=["Date", "Score"])
+
+df["Date"] = pd.to_datetime(df["Date"], format="%Y%m%d")
+
+fig = pygmt.Figure()
+region = pygmt.info(
+    table=df[["Date", "Score"]],
+    per_column=True,
+    spacing=(5000, 1200),
+)
+
+fig.plot(
+    region=region,
+    projection="X15c/10c",
+    frame=["WSen", "afg"],
+    x=df.Date,
+    y=df.Score,
+    style="c0.4c",
+    pen="1p",
+    color="green3",
+)
+fig.show()
+
+########################################################################################
+# Setting Primary and Secondary Time Axes
+# ----------------------
+#
+# This example focuses on labeling the axes and setting intervals
+# at which the labels are expected to appear. All of these modification
+# are added to the ``frame`` argument and each item in that list modifies
+# a specific section of the plot.
+#
+# Starting off with ``WS``, adding this string means that only
+# Western/Left **(W)** and Southern/Bottom **(S)** borders of
+# the plot will be shown. For more information on this, please
+# refer to :docs:`pygmt.Frames`.
```

Comment:
I'm not sure why this is not being built correctly

Reply:
What do you mean with "not being built correctly" ? For me the plot looks as I would expect it.

Reply:
I was expecting 
```
# refer to :docs:`pygmt.Frames\`
```
To be shown similar to this where I can click on it.
![Screenshot from 2021-04-16 09-33-05](https://user-images.githubusercontent.com/55197145/115032179-08cf8480-9e97-11eb-96d9-71003c5d166c.png)

However, this is what it showed up as, clicking on it didn't take me to the right page:
![Screenshot from 2021-04-16 09-32-41](https://user-images.githubusercontent.com/55197145/115032272-213f9f00-9e97-11eb-8db0-57f022d5eeed.png)

Reply:
``pygmt.Frames`` does not exist as a module, so to what would you like to reference at this point?

Reply:
I was trying to reference this page: https://www.pygmt.org/latest/tutorials/frames.html

Reply:
Please try  `` :doc:`frame instructions </tutorials/frames>` ``. Here you can replace **frame instructions** by what ever you want to be displayed.